sound/soc/intel/boards/ehl_rt5660.c

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ * ehl_rt5660 - ASOC Machine driver for Elkhart Lake platforms
+ * with rt5660 codec
+ */
+
+#include <linux/module.h>
+#include <linux/acpi.h>
+#include <sound/core.h>
+#include <sound/jack.h>
+#include <sound/pcm.h>
+#include <sound/pcm_params.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+
+#include "hda_dsp_common.h"
+#include "../../codecs/rt5660.h"
+
+#define RT5660_CODEC_DAI	"rt5660-aif1"
+#define DUAL_CHANNEL 2
+
+#define IDISP_CODEC_MASK	0x4
+
+struct sof_card_private {
+	struct list_head hdmi_pcm_list;
```

Comment:
`#include <linux/list.h>`

Reply:
@lyakh Thanks for your reviews.
I'm not sure whether this is necessary or recommended. I checked some other files, such as soc-core.c and it shows soc-core.c and other files don't include such header files explicitly if not necessary.